regression-test/data/nereids_p0/json_p0/test_json.csv

```diff
@@ -0,0 +1,25 @@
+1	\N
+2	null
+3	true
```

Comment:
add all jsonb function test in regression-test/suites/nereids_function_p0/scalar_function